pki/certificates_ext_test.go

```diff
@@ -22,6 +22,7 @@ import (
 )
 
 func TestCertificateSigningWorks(t *testing.T) {
+	t.Skip("persistence not implemented")
```

Comment:
Any reason why these aren't removed instead?

Reply:
Not really, I've kept the `pki` package for now since it's at least isolated from everything else, but it is also not used. So we could remove the `pki` package as well.